docs/UserGuide.md

```diff
@@ -342,75 +342,97 @@ A success message will be displayed if the suppliers are successfully sorted.
 
 ### Adding a delivery: `add -d`
 
-Adds a delivery to the address book.
+Adds a delivery to VendorVault.
 
-Format: `add -d on/DELIVERY_DATE_TIME s/SUPPLIER_INDEX pro/PRODUCT q/QUANTITY kg/g/L/mL/units c/COST`
+Format: `add -d on/DELIVERY_DATE_TIME s/SUPPLIER_INDEX pro/PRODUCT q/QUANTITY c/COST`
+
+<box type="details" seamless>
 
 Parameters:
 
-- `on/DELIVERY_DATE_TIME`: Must be in dd-MM-yyyy hh:mm format, and must not be blank.
-- `s/SUPPLIER_INDEX`: Must be a number greater than 0, and must not be blank.
-- `pro/PRODUCT`: Must be alphanumeric, can include spaces but must not start with a space, and must be between 1 and 50 (inclusive) characters long.
-- `q/QUANTITY`: Must be a number greater than 0 followed by a space, and unit and must not be blank.
-- `c/COST`: Must be a number greater than 0 with up to 2 decimal places allowed, and must not be blank.
-<box type="tip" seamless>
+- `on/DELIVERY_DATE_TIME`: `DELIVERY_DATE_TIME` is the date and time of delivery. It must be in dd-MM-yyyy hh:mm format, and cannot be blank.
+- `s/SUPPLIER_INDEX`:`SUPPLIER_INDEX` is the index of supplier currently displayed. It must be a number between 1 and the total number of suppliers currently displayed (inclusive), and cannot be blank.
+- `pro/PRODUCT`: `PRODUCT` is the product associated with the delivery. It must be alphanumeric, only contain between 1 and 50 (inclusive) characters, and spaces are also allowed.
+    - One space is counted as one character.
+    - `PRODUCT` cannot be made up of only spaces.
+- `q/QUANTITY`: `QUANTITY` is the amount of product to be delivered with units. It must be a number greater than 0 followed by a space, and a unit and must not be blank.
+    - `QUANTITY` cannot have decimal places.
+    - `QUANTITY` units are case sensitive.
+- `c/COST`: `COST` is the total cost for the delivery. It must be a number greater than 0 with up to 2 decimal places allowed, and cannot be blank.
+
+**Tips:** 
+- Day and month of DELIVERY_DATE_TIME must be in double digits!
 
-**Tip:** Day and month of date must be in double digits!
 </box>
 
 <box type="warning" seamless>
 
 **Warnings**:
-- A spacing between `add` and `-d` is compulsory
-- A delivery is considered duplicate if they have the same DATE, TIME, SUPPLIER, PRODUCT and COST
-- Duplicate delivery will not be added again
-- No duplicate prefix can be used
-- Units for QUANTITY is case-sensitive
+- At least one space is needed between `add` and `-d`.
+- A warning will be given if the user tries to add a duplicate delivery.
+- A delivery is considered duplicate and will not be added again if it has the same `DELIVERY_DATE_TIME`, `SUPPLIER`, `PRODUCT`, `QUANTITY`, `COST` and `STATUS` as an existing delivery.
+    - Comparison between different `PRODUCT`is case-sensitive.
+- A delivery has a default `STATUS` of `PENDING`.
+
```

Comment:
Would it be better to also mention that 
**A space between a parameter and the next prefix is compulsory**?

I think we should also mention that adding a delivery for an inactive supplier is not allowed an will raise an error message

Reply:
Yup i think these are good points,  will edit the ug to include these points